packages/components/psammead-navigation/src/index.jsx

```diff
@@ -159,6 +170,7 @@ const StyledNav = styled.nav`
 
 const Navigation = ({ children, dir }) => (
   <StyledNav role="navigation" dir={dir}>
+    <CanonicalHamburgerMenu announcedText="Menu" onOpen={() => {}} />
```

Comment:
This PR looks really good! Only thing is I don't think we want to compose the menu into this export (or have one 'Navigation' export anymore in fact) - I think we'll want to compose the navigation together in simorgh? We can then create the `onOpen` function in Simorgh and pass it into the hamburger menu, and use the AMP or Canonical specific components as required.

Reply:
That's a good point to consider. Though if we decide to compose navigation in Simorgh, then we would need to export a lot of components here and it might get a bit messy?

With regards to Canonical/AMP, if we decided to leave the Navigation as is, we could pass a prop and depending on it render canonical or amp burger menu.

Also, if we compose the nav in Simorgh, this is something that will affect @DenisHdz too for the swipeable navigation.

Reply:
Let's chat today after standup with @DenisHdz and make sure we're all on the same page :)